src/rest/Ratings/addRatingToPlace.ts

```diff
@@ -0,0 +1,141 @@
+import type { Request, Response } from "express";
+import { addRating } from "../../db/Rating/rating";
+import { ObjectId } from "mongodb";
+import type { Rating } from "../../db/types";
+import StatusCode from "../status";
+import {
+	AccountNotFoundErrorJSON,
+	isAuthenticated,
+	MissingParametersErrorJSON,
+	PlaceDoesNotExistErrorJSON,
+	placeExists,
+	RatingAlreadyExistsErrorJSON,
+	RatingCreatedJSON,
+	UnauthorizedErrorJSON,
+	userExists,
+	WrongParamatersErrorJSON,
+} from "../../util";
+import { DbOperationError } from "../../types";
+import { handleError } from "../../util";
```

Comment:
```suggestion
import {
	AccountNotFoundErrorJSON,
	isAuthenticated,
	MissingParametersErrorJSON,
	PlaceDoesNotExistErrorJSON,
	placeExists,
	RatingAlreadyExistsErrorJSON,
	RatingCreatedJSON,
	UnauthorizedErrorJSON,
	userExists,
	WrongParamatersErrorJSON,
	handleError
} from "../../util";
import { DbOperationError } from "../../types";
```
Shouldn't this be imported up here?

Reply:
yeah, good catch